x-pack/plugins/ml/public/embeddables/anomaly_charts/anomaly_charts_embeddable.tsx

```diff
@@ -40,12 +41,25 @@ export class AnomalyChartsEmbeddable extends AnomalyDetectionEmbeddable<
   private reload$ = new Subject<void>();
   public readonly type: string = ANOMALY_EXPLORER_CHARTS_EMBEDDABLE_TYPE;
 
+  // API
+  public entityFields = new BehaviorSubject<MlEntityField[] | undefined>(
```

Comment:
Why not use `embeddableOutputToSubject`?

Reply:
Made it for one embeddable and forgot the other 🤦🏻 Updated in https://github.com/elastic/kibana/pull/178729/commits/d4debca417ad190c407135df4c619c102944c9ee